CONTRIBUTING.md

```diff
@@ -34,6 +34,52 @@ Symfony projects for instance).
 * Check if tests pass: `vendor/bin/phpunit`.
 * Fix project itself: `php php-cs-fixer fix`.
 
+## Working With Docker
+
+This project provides a Docker setup that allows working on it using any of the supported PHP versions.
+
+To use it, you first need to install:
+
+ * [Docker](https://docs.docker.com/get-docker/)
+ * [Docker Compose](https://docs.docker.com/compose/install/)
+
+Make sure the versions installed support [Compose file format 3.8](https://docs.docker.com/compose/compose-file/).
+
+Next, copy [`docker-compose.override.yaml.dist`](./docker-compose.override.yaml.dist) to `docker-compose.override.yaml`
```

Comment:
question: is this step obligatory?

Reply:
Currently yes: the images require the `DOCKER_USER_ID` and `DOCKER_GROUP_ID` arguments to create a guest user that matches the host user to avoid file permission issues. If there's a better alternative, I'd be happy to update.